cfgov/unprocessed/js/organisms/VideoPlayer.js

```diff
@@ -33,174 +20,163 @@ let _stopVideoBinded;
  * @returns {VideoPlayer} An instance.
  */
 function VideoPlayer( element ) {
-
-  // Attach event methods.
-  const eventObserver = new EventObserver();
-  this.addEventListener = eventObserver.addEventListener;
-  this.dispatchEvent = eventObserver.dispatchEvent;
-  this.removeEventListener = eventObserver.removeEventListener;
-
-  _dom = checkDom( element, BASE_CLASS );
-  _videoId = _dom.getAttribute( 'data-id' );
-  _imageDom = _dom.querySelector( `.${ BASE_CLASS }_image` );
-  _defaultImageURL = _imageDom.src;
-  _playLinkDom = _dom.querySelector( `a.${ BASE_CLASS }_play-btn` );
-  _playBtnDom = _dom.querySelector( `button.${ BASE_CLASS }_play-btn` );
-  _closeBtnDom = _dom.querySelector( `.${ BASE_CLASS }_close-btn` );
-  _iframeDom = _dom.querySelector( `.${ BASE_CLASS }_iframe` );
-
-  _playVideoBinded = playVideo.bind( this );
-  _stopVideoBinded = stopVideo.bind( this );
-
-  // Expose the public methods.
-  this.init = init;
-
-  return this;
-}
-
-/**
- * @returns {VideoPlayer} An instance.
- */
-function init() {
-  setInitFlag( _dom );
-
-  // Retrieve the image URL and load the image.
-  let imageURL = '';
-  if ( _videoId !== null ) {
-    imageURL = youTubeAPI.fetchImageURL( _videoId );
+  const _dom = checkDom( element, BASE_CLASS );
+
+  const _videoId = _dom.getAttribute( 'data-id' );
+  const _showCustomThumbnail = _dom.hasAttribute( 'data-custom-thumbnail' );
+
+  const _iframeDom = _dom.querySelector( `.${ BASE_CLASS }_iframe` );
+  const _imageDom = _dom.querySelector( `.${ BASE_CLASS }_image` );
+  const _closeBtnDom = _dom.querySelector( `.${ BASE_CLASS }_close-btn` );
+  const _playBtnDom = _dom.querySelector( `button.${ BASE_CLASS }_play-btn` );
```

Comment:
Hm this should probably be a consistent query to that of the close button. So it should omit `button` here or the close button should include `button`.

Reply:
I guess that `button.` is included here because [the next line](https://github.com/cfpb/cfgov-refresh/blob/c280e5d0b0f10f3ad74508e471565e8960740527/cfgov/unprocessed/js/organisms/VideoPlayer.js#L32) refers to the same classes for a different element type:

```html
  const _playBtnDom = _dom.querySelector( `button.${ BASE_CLASS }_play-btn` );
  const _playLinkDom = _dom.querySelector( `a.${ BASE_CLASS }_play-btn` );
```

Is this correct? Might make sense to add `button.` to the play button, then, but then why not have it on the other selectors?

Reply:
Yeah I'd just add it so the buttons are consistent and not worry about the rest above it. I don't know for sure, but narrowing down the selector by element type may make it microscopically faster to look up.

Reply:
Fixed in latest commit.